varia/functions.php

```diff
@@ -145,19 +145,25 @@ function varia_setup() {
 		// $lightness       = varia_get_default_lightness();
 		// $lightness_hover = varia_get_default_lightness_hover();
 
+		$colors_array = get_theme_mod( 'colors_manager' );
```

Comment:
Minor, but can you add a code comment explaining what this is?

Reply:
Added some comments in a28de42

Reply:
Thanks!